src/Discovery/Oidc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/******************************************************************************
+ * Copyright 2017 Okta, Inc.                                                  *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License");            *
+ * you may not use this file except in compliance with the License.           *
+ * You may obtain a copy of the License at                                    *
+ *                                                                            *
+ *      http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ ******************************************************************************/
+
+namespace Okta\JwtVerifier\Discovery;
+
+use Okta\JwtVerifier\Discovery\DiscoveryMethod as Discovery;
+
+class Oidc extends Discovery
```

Comment:
Splitting this out into a class seems like a cool idea!
I was just going to trigger on a string, but something like this might allow us to do a few more specifics when parsing the differences between the well-know response body